src/scripts/towns/Town.ts

```diff
@@ -517,3 +517,89 @@ TownList['Pinwheel Forest'] = new DungeonTown('Pinwheel Forest', [new GymBadgeRe
+TownList['Ambrette Town'] = new Town('Ambrette Town', [new RouteKillRequirement(10, 8)], AmbretteTownShop);
+TownList['Cyllage City'] = new Town('Cyllage City', [new ClearDungeonRequirement(1, Statistics.getDungeonIndex('Glittering Cave'))]);
+TownList['Geosenge Town'] = new Town('Geosenge Town', [new RouteKillRequirement(10, 10)], GeosengeTownShop);
+TownList['Shalour City'] = new Town('Shalour City', [new ClearDungeonRequirement(1, Statistics.getDungeonIndex('Reflection Cave'))], ShalourCityShop);
+TownList['Coumarine City'] = new Town('Coumarine City', [new RouteKillRequirement(10, 12)], CoumarineCityShop);
+TownList['Laverre City'] = new Town('Laverre City', [new RouteKillRequirement(10, 14)], LaverreCityShop);
+TownList['Dendemille Town'] = new Town('Dendemille Town', [new RouteKillRequirement(10, 15)], DendemilleTownShop);
+TownList['Anistar City'] = new Town('Anistar City', [new RouteKillRequirement(10, 17)], AnistarCityShop);
+TownList['Couriway Town'] = new Town('Couriway Town', [new RouteKillRequirement(10, 18)], CouriwayTownShop);
+TownList['Snowbelle City'] = new Town('Snowbelle City', [new RouteKillRequirement(10, 19)]);
+TownList['Pokémon League Kalos'] = new Town('Pokémon League Kalos', [new ClearDungeonRequirement(1, Statistics.getDungeonIndex('Victory Road Kalos'))]);
+
+//Kalos Dungeons
+TownList['Santalune Forest'] = new DungeonTown('Santalune Forest', [new RouteKillRequirement(10, 2)]);
+TownList['Parfum Palace'] = new DungeonTown('Parfum Palace', [new RouteKillRequirement(10, 6)]);
+TownList['Connecting Cave'] = new DungeonTown('Connecting Cave', [new RouteKillRequirement(10, 7)]);
+TownList['Glittering Cave'] = new DungeonTown('Glittering Cave', [new RouteKillRequirement(10, 9)]);
+TownList['Reflection Cave'] = new DungeonTown('Reflection Cave', [new RouteKillRequirement(10, 11)]);
+//Tower of Mastery?
+TownList['Azure bay'] = new DungeonTown('Azure bay', [new RouteKillRequirement(10, 12)]);
+//Sea Spirit's Den?
+//Kalos Power Plant?
+TownList['Lost Hotel'] = new DungeonTown('Lost Hotel', [new RouteKillRequirement(10, 15)]);
+TownList['Frost Cavern'] = new DungeonTown('Frost Cavern', [new RouteKillRequirement(10, 15)]);
+TownList['Team Flare Secret HQ'] = new DungeonTown('Team Flare Secret HQ', [new GymBadgeRequirement(BadgeCase.Badge.Psychic)]);
+TownList['Terminus Cave'] = new DungeonTown('Terminus Cave', [new RouteKillRequirement(10, 18)]);
+TownList['Pokémon Village'] = new DungeonTown('Pokémon Village', [new RouteKillRequirement(10, 20)]);
+TownList['Victory Road Kalos'] = new DungeonTown('Victory Road Kalos', [
+    new OneFromManyRequirement([
+        new MultiRequirement([
+            new GymBadgeRequirement(BadgeCase.Badge.Iceberg),
+            new RouteKillRequirement(10, 21),
+        ]),
+        new MultiRequirement([
+            new GymBadgeRequirement(BadgeCase.Badge.Iceberg),
+            new RouteKillRequirement(10, 22),
+        ]),
+    ]),
```

Comment:
I think this should be doing the same thing, as by default the array is basically a `MultiRequirement`
```suggestion
    new GymBadgeRequirement(BadgeCase.Badge.Iceberg),
    new OneFromManyRequirement([
        new RouteKillRequirement(10, 21),
        new RouteKillRequirement(10, 22),
    ]),
```